src/Dependencies/Collections/Internal/IDictionaryDebugView`2.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.Collections.Internal
+{
+    internal sealed class IDictionaryDebugView<K, V>
```

Comment:
Should not that this file comes from dotnet/runitme and future updates should be replicated here.

Reply:
Done